sklearn/linear_model/coordinate_descent.py

```diff
@@ -1063,6 +1063,8 @@ def fit(self, X, y):
 
         if hasattr(self, 'l1_ratio'):
             model_str = 'ElasticNet'
+            if self.l1_ratio == 0:
+                raise ValueError("l1_ratio = 0 not supported")
```

Comment:
Maybe you could add a mention about L2 penalty and RidgeCV like you did in the docstring?

Just curious what happens if l1_ratio is a list which contains 0?